tasks/eth/008-presigned-pause/README.md

```diff
@@ -0,0 +1,53 @@
+# Superchain Presigned Pause
+
+Status: DRAFT
+
+Set the `SCRIPT` env var with the following command:
+
+```
+export SCRIPT=PresignPauseFromJson_eth_008
+```
+
+Then see [../../../PRESIGNED-PAUSE.md](../../../PRESIGNED-PAUSE.md) for the remainder of the
+playbook.
+
+## Additional Overrides
```

Comment:
It seems the regular overrides on 0x9BA6e03D8B90dE867373Db8cF1A58d2F7F006b3A (the foundation operations safe) are never explained here or in PRESIGNED-PAUSE.md

Reply:
> You will see a state override (not a state change). This is expected and its purpose is to generate a successful Safe execution simulation without collecting any signatures.

This is what `PRESIGNED-PAUSE.md` says, but there are 4 overrides

Reply:
Yes, I was operating on the basis that if that was sufficient in the past it should be sufficient now.

Reply:
I went ahead and added them to the README in this dir.